frontend/src/components/teacher/student-management/classrooms/ClassroomCard.tsx

```diff
@@ -28,6 +28,8 @@ interface ClassroomCardProps {
   assessmentCount: number;
   grade: Grade;
   activeAssessments: number;
+  disabledOptions?: boolean;
```

Comment:
maybe rename to something more descriptive like `clickDisabled`? or were there other event actions you had in mind under "options"?